docs/src/components/Navigation/NavigationLayout.tsx

```diff
@@ -24,6 +24,7 @@ export const NavigationLayout = ({ navigation }: NavigationLayoutProps) => {
             <Button
               variant="navigationSmall"
               onPress={() => setShowNavigation(!showNavigation)}
+              title="navigation"
```

Comment:
should this be an `aria-label` 🤔

Reply:
oh you are right. I will add aria-label
![image](https://user-images.githubusercontent.com/27921741/188629960-58dc590e-d6cb-4ced-9af3-4eb51795ef02.png)